<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarTarea</class>
 <widget class="QMainWindow" name="AgregarTarea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Tarea</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID del Objeto:</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblidObj">
        <property name="text">
         <string>código</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Materia:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboMat"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Descripción:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtDescrp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Grupo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkGrp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Prioridad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnPrio"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Porcentaje:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spnPorc">
            <property name="maximum">
             <double>100.989999999999995</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Entregar: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtEnt">
            <property name="displayFormat">
             <string notr="true">dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbERR">
            <property name="text">
             <string>RR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Adelantar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dtAd">
            <property name="displayFormat">
             <string notr="true">dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbARR">
            <property name="text">
             <string>RR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="btnAccept">
        <property name="text">
         <string>Aceptar</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/rescr/ing_accept</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/rescr/cancell</normalon>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFix_Screen"/>
   <addaction name="actionH_img"/>
  </widget>
  <action name="actionFix_Screen">
   <property name="icon">
    <iconset resource="rsour.qrc">
     <normaloff>:/rescr/UpdateIcon</normaloff>:/rescr/UpdateIcon</iconset>
   </property>
   <property name="text">
    <string>Fix Screen</string>
   </property>
  </action>
  <action name="actionH_img">
   <property name="icon">
    <iconset resource="rsour.qrc">
     <normaloff>:/rescr/mov_obj</normaloff>:/rescr/mov_obj</iconset>
   </property>
   <property name="text">
    <string>H_img</string>
   </property>
   <property name="toolTip">
    <string>View Horary</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsour.qrc"/>
 </resources>
 <connections/>
</ui>
